biscuit-auth-datalog-macros/tests/simple_test.rs

```diff
@@ -0,0 +1,96 @@
+extern crate biscuit_auth;
+extern crate biscuit_quote;
+use biscuit_auth::KeyPair;
+use biscuit_quote::{authorizer, biscuit, block};
+
+#[test]
+fn block_macro() {
+    let b = block!(
+        r#"fact("test", hex:aabbcc, [true], {my_key});
+            rule($0, true) <- fact($0, $1, $2, {my_key});
+            check if {my_key}.starts_with("my");
+            "#,
+        my_key = "my_value",
+        other_key = false,
+    );
+    assert_eq!(
+        format!("{}", &b),
```

Comment:
```suggestion
        format!("{}", &b),
```
```suggestion
        b.to_string(),
```

If `Display` is implemented, you get `ToString` too